build_config.rb

```diff
@@ -22,7 +22,8 @@
   conf.gem :mgem => "mruby-onig-regexp"
   conf.gem :mgem => "mruby-method"
   conf.gem :mgem => "mruby-catch-throw"
-  conf.gem :github => "kou/mruby-pp"
+  conf.gem :github => 'haconiwa/mruby-exec'
```

Comment:
Yes, it's going to be much more practical to use other commands as well.
But maybe we can avoid the exec if we can see that the target ruby is the same.